gradle.properties

```diff
@@ -1,4 +1,4 @@
-quasarVersion=0.9.0_r3-SNAPSHOT
+quasarVersion=0.9.1_r3-SNAPSHOT
```

Comment:
There is no 0.9.0_r3 release of Quasar yet, so can you revert this please?

Reply:
reverted